### PR TITLE
Move all services in a single package

### PR DESCRIPTION
**Pull request by palango**
This will make maintenance a lot easier.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
 # until a stable version with handle_exception is released on pip
 git+https://github.com/matrix-org/matrix-python-sdk.git
 
 git+https://github.com/raiden-network/raiden-contracts.git
-git+https://github.com/raiden-network/raiden-libs.git
+# raiden-contracts==0.8.0
 
-gevent
-jsonschema
+dataclasses==0.6
```
Comment (karlb) [inline requirements.txt]: Not really related to this PR: Raiden just moved to python3.7. If we follow suit, we won't need this as a dependency, anymore.

File: src/raiden_libs/utils/signing.py [hunk 2]
```diff
@@ -138,21 +137,11 @@ def eth_sign(
         v: int = 27,
         hasher: Hasher = eth_sign_sha3,
 ) -> bytes:
-    warnings.warn(
```
Comment (karlb) [inline src/raiden_libs/utils/signing.py]: Should we remove these warnings? It sounds like we should remove these functions and call functions from `raiden`, instead.

File: requirements.txt (hunk 1 above)
Comment (palango) [reply requirements.txt]: Good point. The porblem is that there is no python 3.7 environment on travis yet. So either we switch to CircleCI as well ort find another solution, but I agree we should use 3.7 as well if Raiden does it.

File: src/raiden_libs/utils/signing.py (hunk 2 above)
Comment (palango) [reply src/raiden_libs/utils/signing.py]: I was mostly annoyed by the warning, but I can remove the commit before merging

File: requirements.txt (hunk 1 above)
Comment (palango) [reply requirements.txt]: Opened #53

**Pull request merged**